### ISSUE TITLE
No module named 'imghdr'

### ISSUE THREAD

**Issue by cherpake** [postlund/pyatv#2529]
### Describe the bug

No module named 'imghdr'... 
Seems it was deprecated and removed https://github.com/sphinx-doc/sphinx/issues/10440

### Error log

```log
Traceback (most recent call last):
  File "/Users/cherpake/.local/bin/atvremote", line 5, in <module>
    from pyatv.scripts.atvremote import main
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/__init__.py", line 26, in <module>
    from pyatv.protocols import PROTOCOLS
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/protocols/__init__.py", line 10, in <module>
    from pyatv.protocols import airplay as airplay_proto
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/protocols/airplay/__init__.py", line 29, in <module>
    from pyatv.protocols.airplay.ap2_session import AP2Session
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/protocols/airplay/ap2_session.py", line 23, in <module>
    from pyatv.support.rtsp import RtspSession
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/support/rtsp.py", line 18, in <module>
    from pyatv.support.metadata import MediaMetadata
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/pyatv/support/metadata.py", line 7, in <module>
    from mediafile import MediaFile
  File "/Users/cherpake/.local/pipx/venvs/pyatv/lib/python3.13/site-packages/mediafile.py", line 52, in <module>
    import imghdr
ModuleNotFoundError: No module named 'imghdr'
```

### How to reproduce the bug?

Install python3.13

### What is expected behavior?

Some kind of fix

### Operating System

macOS

### Python

Other

### pyatv

latest

### Device

any

### Additional context

That's it

**Comment by postlund** [comment]
Bummer! Seems to be blocked by this:

https://github.com/beetbox/mediafile/issues/67

**Comment by cherpake** [comment]
Maybe there is an alternative to this lib. What’s it used for in pyatv?Sent from my iPhoneOn 22 Oct 2024, at 19:57, Pierre Ståhl ***@***.***> wrote:﻿
Bummer! Seems to be blocked by this:
beetbox/mediafile#67

**Comment by postlund** [comment]
There probably is, I just found mediafile very convenient to use. It's just only used extract some metadata from audio files, e.g. artist, title and such. So it should be fairly easy to replace.

**Comment by postlund** [comment]
This one is a contender: https://github.com/tinytag/tinytag

**Comment by postlund** [comment]
I'm working on migrating to tiny tag, will try to finish off of this weekend.

**Comment by postlund** [comment]
This should be fixed by #2532, which is merged now. I have a few more things I want to get merged, so I will fix that and make a release once everything is ready. Let's see if I can make it within the next few days.
